src/cobase/components/organisms/Reshaping/style.css

```diff
@@ -0,0 +1,42 @@
+@import 'colors.css';
+@import 'typography.css';
+
+.cobase-reshaping {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 96px;
+  margin-bottom: 119px;
```

Comment:
`padding-top: 96px;` and remove `margin-bottom` here
You should add `padding-top: 119px;` to `.mobile` class